code/modules/medical/cloner_defects.dm

```diff
@@ -247,28 +277,36 @@ ABSTRACT_TYPE(/datum/cloner_defect/missing_organ)
 	severity = CLONER_DEFECT_SEVERITY_MAJOR
 
 	init()
-		src.data = list("organ_string" = pick("liver", "left_eye", "right_eye", "left_lung", "right_lung")) // no skull, head, brain; instant death sucks. also removed heart
+		src.data = list("organ_string" = pick("left_eye", "right_eye", "left_lung", "right_lung", "liver")) // no skull, head, brain; instant death sucks. also removed heart
+
+/datum/cloner_defect/missing_organ/heart
+	severity = CLONER_DEFECT_SEVERITY_MAJOR
+	weight = 10
```

Comment:
This seems heavily reliant on an active and available medbay. What is the expected life expectancy for this? 

This seems like a major bummer for a new player to bump into.

Reply:
Admittedly extremely rare occurrence, but the concern is warranted. A lot of these were made with a doctor on stand-by in mind, or at the very least someone with medical knowledge, partially to deter random passer-by cloner usage.

If it's a major concern for new players, we could exclude newbees from contracting it.

Reply:
> partially to deter random passer-by cloner usage

I'd prefer that the behavior can account for the current health/status/population of the server or is going to be conducive to providing agency to players.  

> The intent is for cloning to be a last resort that should be used if the patient is already dead, and encourages having someone with medical knowledge nearby that can assist the person, but does so in a way that doesn't require it.

Doesn't this require someone with medical knowledge to be nearby and assist?

Reply:
Yeah, I'm convinced. We can exclude it from normal selection and perhaps make it emag-only or something.